tests/test_csc.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+
+from lsst.ts import salobj, electrometer
+import asynctest
```

Comment:
Usually I am asked to order the imports such that standard modules and of non-standard imports are separated by a blank line. I'll leave it to you to decide whether you want to reorder the imports and introduce a blank line or not.